src/webhooks/test/registry.test.ts

```diff
@@ -397,10 +385,10 @@ describe('ShopifyWebhooks.Registry.register', () => {
       topic: 'PRODUCTS_CREATE',
       accessToken: 'some token',
       shop: 'shop1.myshopify.io',
-      webhookHandler: genericWebhookHandler,
     };
+    await ShopifyWebhooks.Registry.addHandler({path: '/webhooks', topic: 'PRODUCTS_CREATE', webhookHandler: genericWebhookHandler});
     await ShopifyWebhooks.Registry.register(webhook);
-    expect(ShopifyWebhooks.Registry.webhookRegistry).toHaveLength(1);
+    expect(Object.keys(ShopifyWebhooks.Registry.webhookRegistry).length).toBe(1);
```

Comment:
Totally a nit, but you could do this:
```suggestion
    expect(Object.keys(ShopifyWebhooks.Registry.webhookRegistry)).toHaveLength(1);
```

Reply:
this is so much nicer